app/services/audit/replication_to_gcp.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+module Audit
+  # Methods for auditing the state of a ZippedMoabVersion on an IBM S3 compatible endpoint.
```

Comment:
Is GCP IBM S3-compatible?

Reply:
more copypasta, and that should probably just get changed to "S3 compatible" with no vendor reference anyway.  since so far, everything is S3 compatible, with just a bit of extra (URL for the endpoint) config for non-AWS vendors.